tests/vitest.ts

```diff
@@ -6,6 +6,6 @@ export async function test(options: RunOptions) {
 		...options,
 		repo: 'vitest-dev/vitest',
 		build: 'build',
-		test: 'test:run'
+		test: 'test:run -- --allowOnly'
```

Comment:
I don't think you need the additional `--`. That's part of the `pnpm` call in the other test scripts AFAIK.

Reply:
Thanks! Working now 🎉